app/api/trades/route.tsx

```
@@ -0,0 +1,27 @@
+// app/api/trades/route.ts
+import { NextResponse } from "next/server";
+import { tradeEntrySchema } from "@/lib/schemas/trade";
+import fs from "fs/promises";
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)